CogniteSdk/test/fsharp/Groups.fs

```diff
@@ -0,0 +1,63 @@
+﻿module Tests.Integration.Groups
+
+open FSharp.Control.Tasks
+open Swensen.Unquote
+open Xunit
+
+open CogniteSdk
+open Common
+
+[<Fact>]
+let ``List own groups is OK`` () = task {
+    // Act
+    let! res = writeClient.Groups.ListAsync()
+
+    test <@ Seq.length res > 0 @>
+    test <@ Seq.exists (fun (g: Group) ->
+        Seq.exists (fun (cap: BaseAcl) -> cap :? GroupsAcl) g.Capabilities
+    ) res @>
+}
+
+[<Fact>]
+let ``List all groups is OK`` () = task {
+    // Act
+    let! res = writeClient.Groups.ListAsync true
+
+    test <@ Seq.length res > 1 @>
+    test <@ Seq.exists (fun (g: Group) -> g.Name = "admin") res @>
+}
+
+[<Fact>]
+let ``Create delete group is OK`` () = task {
+    // Arrange
+    let capabilities : BaseAcl list = [
+        GroupsAcl(CurrentUserScope=BaseScope(), Actions=["LIST"]);
+        RelationshipsAcl(All=BaseScope(), Actions=["READ"]);
+        RawAcl(
+            TableScope=RawTableScope(
+                DbsToTables=dict["sdk-test-database", RawTableScopeWrapper(Tables=["sdk-test-table"])]
+            ),
+            Actions=["LIST"]
+        )
+    ]
+    let group = GroupCreate(
```

Comment:
Need to put GroupGreate on the next line to avoid indentation warnings since the arguments should not be less indented than the constructor